components/Navbar.js

```diff
@@ -22,6 +22,7 @@ const AUTH_SIGNEDIN_ROUTES = [
   { title: "Hours", route: "/hours" },
   { title: "Order", route: "/order"},
   { title: "Checkout", route: "/checkout"},
+  { title: "Orders", route: "/orders"}
```

Comment:
could we change the display name on the navbar (to "bag packing" or something)? i feel like it could be confusing having `order` and `orders` next to each other

keeping the actual link is fine, just the title maybe?